src/app/src/test/kotlin/org/vaccineimpact/api/tests/models/CreateUserTests.kt

```diff
@@ -19,27 +19,27 @@ class CreateUserTests : ValidationTests()
     @Test
     fun `username must be valid`()
     {
-        assertCausesTheseErrors(model.copy(username = ""), "invalid-field:username:blank")
-        assertCausesTheseErrors(model.copy(username = "user name"), "invalid-field:username:bad-format")
-        assertCausesTheseErrors(model.copy(username = "User.Name"), "invalid-field:username:bad-format")
-        assertCausesTheseErrors(model.copy(username = "user-name"), "invalid-field:username:bad-format")
-        assertCausesTheseErrors(model.copy(username = "user_name"), "invalid-field:username:bad-format")
-        assertCausesTheseErrors(model.copy(username = "12345"), "invalid-field:username:bad-format")
-        assertCausesTheseErrors(model.copy(username = "name&"), "invalid-field:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = ""), "invalid-field:create_user:username:blank")
+        assertCausesTheseErrors(model.copy(username = "user name"), "invalid-field:create_user:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = "User.Name"), "invalid-field:create_user:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = "user-name"), "invalid-field:create_user:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = "user_name"), "invalid-field:create_user:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = "12345"), "invalid-field:create_user:username:bad-format")
+        assertCausesTheseErrors(model.copy(username = "name&"), "invalid-field:create_user:username:bad-format")
```

Comment:
I made the field name more qualified in the error codes, to distinguish `create_burden_estimate_set:type` from `burden_estimate_type_code: type` but it does make these ones perhaps less clear? Maybe we could qualify them only for nested properties.

Reply:
Do you need the top level object? Couldn't it just be `invalid-field:type:type:missing` and `invalid-field:username:blank`